mlrun/launcher/base.py

```diff
@@ -18,7 +18,12 @@
 import uuid
 from typing import Any, Dict, List, Optional, Union
 
+import IPython
```

Comment:
this would require ipython on both server and client - is that acceptable? i think worth moving to specific side or nest within the correct `if` and locally import it

Reply:
moved to client based launcher